src/conn/conn_compact.c

```diff
@@ -338,7 +349,6 @@ __compact_server(void *arg)
     int64_t id;
     bool full_iteration, running, signalled, release_dhandle;
```

Comment:
Do we still need to declare the `release_dhandle` variable?

Reply:
Fixed https://github.com/wiredtiger/wiredtiger/pull/9541/commits/447fd8c07bcd9cdee97983994b076575d231d1f1.